CodeiumVS/Commands.cs

```diff
@@ -318,7 +318,6 @@ protected override async Task ExecuteAsync(OleMenuCmdEventArgs e)
         // highlight the selected codeblock
         TextHighlighter? highlighter = TextHighlighter.GetInstance(docView.TextView);
         highlighter?.AddHighlight(start_position, end_position - start_position);
-
```

Comment:
Eventually we should add a formatter to prevent these whitespace changes from making it in